### PR TITLE
[Backport][10.x] sg/fix/1286430

### PR DESCRIPTION
**Pull request by jessebarker**
### Purpose of this PR
The system call uses the local environment newline which can cause
problems when sharing shadergraph files between Windows and other
systems.  This makes the newline explicitly "\n".

https://fogbugz.unity3d.com/f/cases/1286430/

New ShaderGraph shaders created on Windows had CR/LF line endings for certain scopes. This could cause file difference detection to engage on an unmodified shader file when shared between users on different platforms. This PR fixes that.
---
### Testing status
See https://github.com/Unity-Technologies/Graphics/pull/2483

---
### Comments to reviewers
Simple backport

### CODE REVIEW

File: com.unity.shadergraph/CHANGELOG.md [hunk 1]
```diff
@@ -121,8 +130,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Updated legacy COLOR output semantic to SV_Target in pixel shader for compatibility with DXC.
 - Updated the functions in the `Normal From Height` node to avoid NaN outputs.
 - Changed the Voronoi Node algorithm to increase the useful range of the input values and to always use float values internally to avoid clipping.
-- Changed the `Reference Suffix` of Keyword Enum entries so that you cannot edit them, which ensures that material keywords compile properly. 
-- Updated the dependent version of `Searcher` to 4.2.0. 
+- Changed the `Reference Suffix` of Keyword Enum entries so that you cannot edit them, which ensures that material keywords compile properly.
```
Comment (cdxntchou) [inline com.unity.shadergraph/CHANGELOG.md:133]: What are all of these changes to the changelog?  Did your editor clean up newlines?

**Review by cdxntchou — APPROVED** [review]

File: com.unity.shadergraph/CHANGELOG.md (hunk 1 above)
Comment (jessebarker) [reply com.unity.shadergraph/CHANGELOG.md:133]: Yup.  It's the same thing the conversion will do.

Comment (jessebarker) [reply com.unity.shadergraph/CHANGELOG.md:133]: (they were in the master PR as well)

Comment (cdxntchou) [reply com.unity.shadergraph/CHANGELOG.md:133]: Yeah I saw.   Oh well  :)   Looks good!

**Comment by jessebarker** [comment]
All-shadergraph passes (yay!)

**Pull request merged**